features/create-note.feature

```diff
@@ -24,6 +24,15 @@ Feature: Creating a note
     Then "Note" is in our Outbox
     And "Note" has the content "<p>Hello<br />World</p>"
 
+  Scenario: Created note contains newlines
```

Comment:
Isn't this the same as the scenario above?

Reply:
Yes - I added this to the `create-reply.feature` tests instead